lutris/gui/widgets/default_path.py

```diff
@@ -0,0 +1,129 @@
+import os
+from enum import Enum, auto
+from gettext import gettext as _
+
+from lutris.util.log import logger
+
+
+class PATH_TYPE(Enum):
+    """Types of paths, UNKNOWN should be used by default"""
+
+    UNKNOWN = 0
+    BANNER = auto()
+    ICON = auto()
+    CACHE = auto()
+    INSTALLER = auto()
+    INSTALL_TO = auto()
+
+
+class default_path_handler:
+    """Handles finding the correct path to set in the file chooser dialog"""
+
+    last_selected_path = {PATH_TYPE.UNKNOWN: None}
+
+    @staticmethod
+    def is_valid(path):
+        """`is_valid` Checks if the path is valid
```

Comment:
Why are you adding the function's name in quotes in it's own docstring?

Reply:
It is a style I've gotten used to when documenting internal APIs.  In the past I've used tools to generate html or other such files from internal documentation and had to go back adding that in.  Since I was attempting to put together a single template for everyone, I wanted it to be as forgiving as possible until a particular tool was picked for generating internal tool documentation.

I mean the cleanup is to look for """`[^`]+`\s+ and replace with """ where as putting it in place is much harder.